src/tests/end-to-end/tests/details-view/reflow-ui.test.ts

```diff
@@ -127,6 +127,34 @@ describe('Details View ->', () => {
             });
         });
 
+        describe.each([true, false])('With high contrast mode=%s', highContrastMode => {
```

Comment:
There was an automated check issue that was being missed in fastpass without these tests.

Effectively, we were loading empty divs with role set to menuitem in FastPass for the load and save assessment buttons. This is because the actual methods would return null BUT we were only checking whether the function was null (which happens in Electron but not in FastPass) and not the return value of the function.

The command-bar-buttons-menu has been modified to take in the JSX.Elements instead and since the details-view-command-bar was already rendering/generating the JSX.Elements for the non-narrow case, it made sense to also do so for the narrow case.